specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -4800,10 +4803,22 @@
           "multipleOf": 1.0,
           "minimum": 0,
           "description": "Value indicating the age in days after blob creation."
+        },
+        "autoUptier": {
+          "$ref": "#/definitions/AutoUptier",
+          "description": "Automatically up tiers the blob to the account level access tier after the blob is read. This is a generalized implemntation of enableAutoTierToHotFromCool and is mutually exclusive with enableAutoTierToHotFromCool."
```

Comment:
Should "implemntation" be "implemntation"?
BTW, should we mention how to implement this feature in the description? We might should only mention AutoUptier is mutually exclusive with enableAutoTierToHotFromCool.

Reply:
updated description